docs/java-rest/high-level/snapshot/clone_snapshot.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[java-rest-high-snapshot-clone-snapshot]]
+=== Clone Snapshot API
+
+The Clone Snapshot API allows to clone part or all of a snapshot into a new snapshot.
+
+[[java-rest-high-snapshot-clone-snapshot-request]]
+==== Clone Snapshot Request
+
+A `CloneSnapshotRequest`:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/SnapshotClientDocumentationIT.java[clone-snapshot-request]
+--------------------------------------------------
+
+==== Indices to Clone
+
+The `indices` property is used to provide the list of
+indices that should be included in the snapshot:
```

Comment:
provide the list of indices from the source snapshot to be included in the snapshot clone